openloader/resources/crucial_resources/assets/crucial/lang/en_us.json

```diff
@@ -67,5 +67,9 @@
 	"block.architects_palette.chiseled_gold_ore_bricks": "Gold Etched Stone Bricks",
 	"block.architects_palette.chiseled_emerald_ore_bricks": "Emerald Etched Stone Bricks",
 	"block.architects_palette.chiseled_diamond_ore_bricks": "Diamond Etched Stone Bricks",
-	"block.architects_palette.blank_acacia_totem": "Acacia Totem"
+	"block.architects_palette.blank_acacia_totem": "Acacia Totem",
+	
+	"patchouli.guide_book": "Crucial 2 Guide",
+	"patchouli.guide_book.ver": "Dev Version",
```

Comment:
This doesn't work unfortunately, because the build script requires this string to be in the book json.
Putting this back in the book json and updating this file with the latest strings will make this pullable though.